src/shared/components/cancerSummary/CancerSummaryContent.tsx

```diff
@@ -171,14 +177,14 @@ export class CancerSummaryContent extends React.Component<ICancerSummaryContentP
             mutated:"#008000",
             amplified:"#ff0000",
             deleted:"#0000ff",
-            multiple:"#000000",
-            fusion: "#8B00C9",
-            gain: "rgb(255,182,193)",
-            homdel: "rgb(0,0,255)",
-            mrnaExpressionUp: "#FF989A",
-            mrnaExpressionDown: "#529AC8",
-            protExpressionUp: "#FF989A",
-            protExpressionDown: "#E0FFFF",
+            multiple:"#383838",
+            fusion:"#8B00C9",
+            gain:"rgb(255,182,193)",
+            homdel:"rgb(0,0,255)",
```

Comment:
should this be hetloss? deleted above looks like homdel...

Reply:
No, `homdel` shouldn't be `hetloss`.  

The store returns `hetloss` as `deleted`.  Therefore, here, to get the color of `hetloss`, I use the `deleted` key.  Then, on line 109, if the alteration is labeled as `deleted`, I change its value to `hetloss`.